src/core/tsi/s2a/handshaker/s2a_handshaker_client.cc

```diff
@@ -43,27 +43,21 @@ grpc_byte_buffer* s2a_handshaker_client::s2a_get_serialized_start_client() {
   s2a_ClientSessionStartReq_add_application_protocols(
       start_client, upb_strview_makez(kS2AApplicationProtocol), arena.ptr());
   int32_t* tls_versions = s2a_ClientSessionStartReq_resize_tls_versions(
-      start_client, /* len=*/1, arena.ptr());
+      start_client, /*len=*/1, arena.ptr());
   GPR_ASSERT(tls_versions != nullptr);
   tls_versions[0] = s2a_TLS1_3;
   int32_t* tls_ciphersuites = s2a_ClientSessionStartReq_resize_tls_ciphersuites(
-      start_client, /* len=*/options_->supported_ciphersuites().size(),
-      arena.ptr());
+      start_client, options_->supported_ciphersuites().size(), arena.ptr());
   GPR_ASSERT(tls_ciphersuites != nullptr);
-  size_t counter = 0;
-  for (auto ciphersuite : options_->supported_ciphersuites()) {
-    tls_ciphersuites[counter] = ciphersuite;
-    counter += 1;
+  for (size_t i = 0; i < options_->supported_ciphersuites().size(); i++) {
```

Comment:
Can we keep the iterator instead of using a counter `i`?

Reply:
Sure thing - switched it back in the next commit.